src/current/v25.2/create-policy.md

```diff
@@ -66,7 +66,6 @@ The following table shows which policies are applied to which statement types, w
 
 Additional considerations include:
 
-- `SELECT` evaluation: CockroachDB always evaluates `SELECT` (`USING`) policies for `INSERT`, `UPDATE`, and `DELETE`, even when the statement doesn't reference table columns.
```

Comment:
Can we include further details as to when the SELECT policies will be applied during an INSERT/UPDATE/DELETE? SELECT policies are always applied during the fetch of the existing row (for UPDATE or DELETE). If a column is referenced in the WHERE, SET or RETURNING clause, then SELECT policies will also get applied during the write.

I think the postgres docs handled this by adding superscript numbers to the above table as an additional addendum.

Reply:
I've taken a shot at addressing this feedback in https://github.com/cockroachdb/docs/pull/19720/commits/8af8a1fe00061a4709a06a4520c11f3868f4883a 

tried to incorporate the info from your comment and add some superscripts and more explanation 

PTAL!